data/tlg4102/tlg008/tlg4102.tlg008.opp-grc1.xml

```diff
@@ -73,11 +73,11 @@
               <imprint>
                 <publisher>Oxford University Press</publisher>
                 <pubPlace>Oxford</pubPlace>
-                <date>1844</date>
+                <date>1838</date>
```

Comment:
The date in the cts file is different. (1841 vs 1838)

Reply:
Thanks for catching this, some of the dates are wrong and I was trying to fix that. I will fix that now.